pinot-core/src/test/java/org/apache/pinot/queries/BaseQueriesTest.java

```diff
@@ -97,12 +98,12 @@ protected BrokerResponseNative getBrokerResponseForQuery(String query, PlanMaker
    * @return broker response.
    */
   private BrokerResponseNative getBrokerResponseForQuery(String query, PlanMaker planMaker,
-      Map<String, String> queryOptions) {
+      Map<String, String> queryOptions, long timeout) {
     BrokerRequest brokerRequest = COMPILER.compileToBrokerRequest(query);
     brokerRequest.setQueryOptions(queryOptions);
 
     // Server side.
-    Plan plan = planMaker.makeInterSegmentPlan(getSegmentDataManagers(), brokerRequest, EXECUTOR_SERVICE, 10_000);
```

Comment:
By default, this should be `CommonConstants.Server.DEFAULT_QUERY_EXECUTOR_TIMEOUT_MS`

Reply:
done